js/ui/tab_cookies.js

```diff
@@ -367,23 +419,88 @@
       texts: {
         name: cookie.name,
         value: cookie.value
+      },
+      titles: {
+         whitelist: getMsg("whitelistSublistCookie")
       }
     };
   }
 
-  function createDomainObj(domain, cookienum)
+  function createDomainObj(domain, cookienum, whitelist = false)
```

Comment:
Seems like we are not using `whitelist = false` parameter now, but I don't see harm of keeping it here, though a comment would have been useful.

Reply:
Something like one below should be sufficient:
```
  /**
   * Create a Table List Item Structure Object
   * @param {String} domain Domain name
   * @param {Number} cookienum Number of domain cookies
   * @param {Boolean} whitelist specifies whether domain is whitelisted
   */
```